src/System.Private.Interop/src/Shared/WindowsRuntimeMarshal.cs

```diff
@@ -977,8 +977,7 @@ internal class ReaderWriterLockTimedOutException : /*ApplicationException*/ Exce
             {
             }
 
-            /// I borrowed Vance's reader writer lock implementation from his blog as ReaderWriterLockSlim is
-            /// available in System.Core.dll!
```

Comment:
I just looked after noticed that the PR now has conflicts, guessing that you had done just that. Yep, I see. Sounds perfect. Will you do the same in coreclr?